inc/deployfile.class.php

```diff
@@ -403,6 +403,11 @@ public static function showServerFileTree($rand)
 
         echo "</script>";
 
+        echo '<div class="alert alert-info d-flex" role="alert">';
+        echo '<i class="fas fa-exclamation-circle fa-2x me-2"></i>';
+        echo '<p>' . __('Files need to be uploaded to <mark>/files/_plugins/glpiinventory/upload/</mark> folder to be displayed here', 'glpiinventory') . '</p>';
```

Comment:
I guess tags and path should not be part of translated string
```suggestion
        echo '<p>' . sprintf(__('Files need to be uploaded to %s folder to be displayed here', 'glpiinventory'), '<mark>/files/_plugins/glpiinventory/upload/</mark>') . '</p>';
```